src/v/config/configuration.cc

```diff
@@ -436,6 +436,27 @@ configuration::configuration()
       "truncation if any of the replicas has more data than the majority.",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       true)
+  , raft_max_inflight_follower_append_entries_requests_per_shard(
+      *this,
+      "raft_max_inflight_follower_append_entries_requests_per_shard",
+      "Maximum number of append entries requests, that were sent to a node "
+      "from a seastar shard, and that the current node is waiting for to be "
+      "replied. (previously this number was configured per raft group, now it "
+      "is configured per shard)",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1024,
+      {.min = 1, .max = 50000})
+  , raft_max_buffered_follower_append_entries_bytes_per_shard(
+      *this,
+      "raft_max_buffered_follower_append_entries_bytes_per_shard",
+      "The total size of append entry requests that may be cached per shard, "
+      "using Raft buffered protocol. When an entry is cached the leader can "
+      "continue serving requests because the ordering of the cached requests "
+      "cannot change. When the queue is full back pressure is propagated to "
```

Comment:
When the total size of cached requests reaches the set limit, back pressure is applied to ????? to limit throughput.

@mmaslankaprv - Would it be accurate to say something like this? We don't seem to refer to `replicate method caller` elsewhere in the docs so I wondered if there was a more generic way to say this?

Reply:
yeah, maybe to 'throttle producers' ?

Reply:
So this @mmaslankaprv?

When the total size of cached requests reaches the set limit, back pressure is applied to throttle producers.